swagger_spec_validator/validator20.py

```diff
@@ -55,6 +55,10 @@ def validate_apis(apis):
         api_params = api_body.get('parameters', [])
         validate_duplicate_param(api_params)
         for oper_name in api_body:
+            # don't treat parameters that apply to all api operations as
+            # an operation
+            if oper_name == 'parameters':
```

Comment:
As per [2.0](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#fixed-fields-4) `$ref` can also be a sibling, i suppose it would be referenced at this point?

As it is a swagger spec validator, we should disallow if `oper_name` is anything other than the given fixed fields (or are we checking it anywhere else)?